examples/flax/language-modeling/run_t5_mlm_flax.py

```diff
@@ -353,7 +353,8 @@ def _random_segmentation(num_items, num_segments):
             np.random.shuffle(mask_indices)
             first_in_segment = np.pad(mask_indices, [[1, 0]])
             segment_id = np.cumsum(first_in_segment)
-            segment_length = np.asarray(jax.ops.segment_sum(np.ones_like(segment_id), segment_id))
```

Comment:
Removing jax from the data collator leads to a huge speed-up here (from 2s/iteration to 1.5 iterations / s)